example/jobflow/README.md

```diff
@@ -8,47 +8,16 @@ read design at [here](../../docs/design/jobflow).
 
 ### Prerequisites
 
-- docker: `18.06`
 - Kubernetes: >`1.17`
 
 ## startup steps
 
-build image from local
-```bash
-# get volcano and jobflow source code from github
-git clone http://github.com/volcano-sh/volcano.git
-git clone https://github.com/BoCloud/JobFlow.git
-
-# build image beyondcent/jobflow:v0.0.1 from local
-cd JobFlow
-make
-make docker-build
-```
-
-##### deploy JobFlow from [here](https://github.com/BoCloud/JobFlow#deploy)
-```bash
-kubectl apply -f https://raw.githubusercontent.com/BoCloud/JobFlow/main/deploy/jobflow.yaml
-```
-
-##### deploy Volcano from [here](https://volcano.sh/en/docs/installation/#install-with-yaml-files)
+### deploy Volcano from [here](https://volcano.sh/en/docs/installation/#install-with-yaml-files)
 ```bash
 kubectl apply -f https://raw.githubusercontent.com/volcano-sh/volcano/master/installer/volcano-development.yaml
 ```
 
-if cert of `jobflow-webhook-service.kube-system.svc` has expired, generate one to replace it.
-```bash
-# delete expired cert in secrets
-kubectl delete secret jobflow-webhook-server-cert -nkube-system
-
-# use gen-admission-secret.sh register new secret
-cd volcano
-./installer/dockerfile/webhook-manager/gen-admission-secret.sh --service jobflow-webhook-service --namespace kube-system --secret jobflow-webhook-server-cert
-
-# restart jobflow-controller-manager
-kubectl delete pod/jobflow-controller-manager-67847d59dd-j8dmc -nkube-system
-```
-
-##### run jobflow example
+### run jobflow example
 ```bash
 # deploy jobTemplate first
 cd volcano
```

Comment:
Maybe provide a hyperlink of the example in line 24 and 26, as the clone operation has been deleted.

Reply:
changed. But I think hyperlink is not necessary. Because this docs and yaml are in the same directory.